docs/misc/deprecated.md

```diff
@@ -15,6 +15,10 @@ The following list of features are deprecated.
 ### Old Command Line Options
 **Deprecated In Release: [v1.8.0](/release-notes/#docker-engine-1-8-0)**
 
+The flags `-d` and `--daemon` are deprecated in favor of the `daemon` subcommand:
+
+    docker daemon -H ...
+
```

Comment:
Hm, you're probably right

(think we should add a migration guide to the release notes - help me remind when the release notes are created)